src/style.css

```
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  margin-inline: 22px;
+  display: flex;
+}
+
+.stars__star {
```

need to use nesting SCSS
```suggestion
.stars {
  margin-inline: 22px;
  display: flex;
  
&__star {
```

---

it doesn't work. I don't use scss in this task. I even didn't study preprocessor yet. Are you sure,?Do I neen to plug-in sass?